### DISCUSSION TITLE
Tailwind classes don't work

### DISCUSSION THREAD

**Discussion by t2ncay** [tailwindlabs/tailwindcss#6208] [Help]
I have problem with Tailwind Create React App project. I did everything on [Tailwind React App setup](https://tailwindcss.com/docs/guides/create-react-app). But when I add some classes like `delay-200`, `bg-blue` and etc., project doesn't apply it. I have no idea why it happens to me.

![image](https://user-images.githubusercontent.com/68643463/143685477-22f0501c-1417-4a37-9c82-b734e51c4c96.png)
 
Any help appreciated, thanks!

**Comment by adamwathan** [comment]
Hey! Can you provide an example project we can look at? Unfortunately working with Create React App is a bit of a nightmare because of how outdated so many of the dependencies are. In general I would recommend using [Vite](https://vitejs.dev/) instead if possible, or compiling your CSS separately using [Tailwind CLI](https://tailwindcss.com/docs/installation#using-tailwind-cli) instead of trying to integrate Tailwind into CRA.

We're going to update the CRA guide soon to recommend the Tailwind CLI approach instead because of how often people run into issues like this.

**Reply by bheemisme** [reply]
Thanks for reply, I have been facing the same issue for a long time

**Reply by t2ncay** [reply]
Yes, I can provide it. [Click here](https://github.com/tuncayofficial/geometry-dash) get source code of the project.

Thanks for your help, I really appreciate it! Also thanks for paying attention to Tailwind CSS's community :slightly_smiling_face:

**Reply by adamwathan** [reply]
So when I clone it it actually is working the way I'd expect:

https://user-images.githubusercontent.com/4323180/143690563-ab6dbe2d-47cd-41bc-9c19-52f1e2e5d851.mov

Am I doing something differently?

**Reply by t2ncay** [reply]
Hello again, no you are not doing something wrong. I reinstalled my packages and it works well now!!! Thanks for trying to help, have a good day sir.
